addons/web/static/src/webclient/actions/client_actions.js

```diff
@@ -12,8 +12,9 @@ export const displayNotificationAction = (env, action) => {
     const options = {
         className: params.className || "",
         sticky: params.sticky || false,
-        title: params.title ? escape(params.title) : "",
+        title: params.title ? params.title : "",
```

Comment:
This simplifies to
```suggestion
        title: params.title || "",
```
which is not even necessary: if `title` is `null` or `undefined` it will simply not be shown by the templating system.

So I believe this can be further simplified to
```suggestion
        title: params.title,
```

Reply:
@xmo-odoo 👍